File: rest/incoming-phone-numbers/instance-post-example-1/instance-post-example-1.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account Sid and Auth Token from https://www.twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Get the accounts with the given sid
@accounts = @client.api.v2010.accounts(account_sid)

# Get incoming phone number with given sid
@number = @accounts.incoming_phone_numbers('PN2a0747eba6abf96b7e3c3ff0b4530f6e')
                   .fetch

# Update the voice and sms urls of the selected incoming number
@number.update(
  voice_url: 'http://demo.twilio.com/docs/voice.xml',
  sms_url: 'http://demo.twilio.com/docs/sms.xml'
)

puts @number.voice_url

File: rest/queue/instance-post-example-1/instance-post-example-1.5.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioRestClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"
client = TwilioRestClient(account_sid, auth_token)

queue = client.queues.update(
    "QU32a3c49700934481addd5ce1659f04d2", max_size="123"
)
print(queue.average_wait_time)
